packages/workflow/src/Extensions/ExpressionExtension.ts

```diff
@@ -11,14 +11,6 @@ import { objectExtensions } from './ObjectExtensions';
 
 const EXPRESSION_EXTENDER = 'extend';
 
-function isBlank(value: unknown) {
-	return value === null || value === undefined || !value;
-}
-
-function isPresent(value: unknown) {
-	return !isBlank(value);
-}
```

Comment:
Removing these will result in `undefined` and `null` not working with `isEmpty`. If this is OK then they're fine to remove but thought I'd point it out if this wasn't intended.

Reply:
@nik8n ^Can you confirm?

Reply:
Discussed and restored. @valya When you can please take another look.